plugins/mysql.yaml

```diff
@@ -1,6 +1,12 @@
-version: 0.0.10
+version: 0.0.11
```

Comment:
Oops seems like we haven't released with 0.0.10, I'll just incorporate my change with that version.

Reply:
Good catch. I had bumped mysql version but haven't released. I had forgotten to bump with last release.